frontend/app/states/play.js

```diff
@@ -79,10 +80,32 @@ class PlayState extends Phaser.State {
       this.game.world.width / 2.5, 0,
       this.game.camera.width / 100, this.game.world.height - this.game.ground.height);
 
-    this.test = new Modal(this.game, new Keyboard(this.game));
+    //add keyboard demo button
+    this.keyboardDemoButton = new Button(this.game, null, 0,
+        this.game.world.height - 135, this.game.camera.width/12,
```

Comment:
We should always avoid hard-coding pixel values like `this.game.world.height - 135` since this will cause unexpected behavior on different viewports. When you need to set x, y, width or height of a sprite, these attributes should always be relative to some in-game object like the world, the camera, or another sprite.

In this instance I think a good value for the y position might be something like:
`this.game.ground.y - this.game.ground.height * 1.1`
which would make it float a little bit above the ground.